.github/workflows/benchmarks.yml

```diff
@@ -67,3 +68,65 @@ jobs:
       - name: Run MT benchmarks
         working-directory: ${{github.workspace}}/build
         run: ./benchmark/multithread_bench
+
+  benchmarks-windows:
+    name: Windows
+    env:
+      BUILD_DIR : "${{github.workspace}}/build/"
+      HWLOC_PATH: "${{github.workspace}}/build/vcpkg/packages/hwloc_x64-windows"
+    strategy:
+      fail-fast: false
+      matrix:
+        os: ['windows-2022']
+        build_type: [Release]
+        compiler: [{c: cl, cxx: cl}]
+        shared_library: ['ON', 'OFF']
+    runs-on: ${{matrix.os}}
+
+    steps:
+    - name: Checkout
+      uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11 # v4.1.1
+
+    - name: Initialize vcpkg
+      uses: lukka/run-vcpkg@v11.5
+      with:
+        vcpkgGitCommitId: 3dd44b931481d7a8e9ba412621fa810232b66289
+        vcpkgDirectory: ${{github.workspace}}/build/vcpkg
+        vcpkgJsonGlob: '**/vcpkg.json'
+
+    - name: Install dependencies
+      run: vcpkg install
+      shell: pwsh # Specifies PowerShell as the shell for running the script.
+
+    - name: Configure build
+      run: >
+        cmake
+        -B ${{env.BUILD_DIR}}
+        -DCMAKE_BUILD_TYPE=${{matrix.build_type}}
```

Comment:
This is ignored on Windows. Passing a `--config` parameter in the build command suffices.